unified-runtime/cmake/FetchLevelZero.cmake

```diff
@@ -63,6 +63,22 @@ if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
     FetchContent_MakeAvailable(level-zero-loader)
     FetchContent_GetProperties(level-zero-loader)
 
+    if(WIN32 AND level-zero-loader_SOURCE_DIR)
+        # apply runtime library patch
```

Comment:
this probably should be under `if(MSVC)`. Also, instead of patching the cmakes, isn't it possible to replace the CXX_FLAGS variables?

Reply:
Done, works locally